src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -265,6 +277,7 @@ protected BoundExpression CreateUserDefinedConversion(SyntaxNode syntax, BoundEx
                 source: userDefinedConversion,
                 conversion: toConversion,
                 isCast: false,
+                isExplicitlyNullable: false,
```

Comment:
It feels like there is something wrong around isExplicitlyNullable handling in this function. For example, why is this the right thing to do to drop isExplicitlyNullable on the final conversion?

Reply:
Added a couple of tests and `// PROTOTYPE` comment to revisit this.